ibm_was/datadog_checks/ibm_was/data/conf.yaml.example

```diff
@@ -0,0 +1,60 @@
+init_config:
+
+instances:
+  ## @param servlet_url - string - required
+  ## The URL of the PerfServlet configured to report metrics
+  #
+  - servlet_url: http://localhost/wasPerfTool/servlet/perfservlet
+
+  ## @param skip_proxy - boolean - optional - default: False
+  ## Wheter or not to skip the proxy configured in the Agent's config file when
+  ## making the request to the PerfServlet 
+  #
+  # skip_proxy: False
+
+  ## @param collect_jdbc_stats - boolean - optional - default: True
+  ## Whether or not to collect JDBC Connection Pool stats
+  #
+  # collect_jdbc_stats: True
+
+  ## @param collect_jvm_stats - boolean - optional - default: True
+  ## Whether or not to collect JVM Runtime stats
+  #
+  # collect_jvm_stats: True
+
+  ## @param collect_servlet_session_stats - boolean - optional - default: True
+  ## Whether or not to collect Servlet Session Manager stats
+  #
+  # collect_servlet_session_stats: True
```

Comment:
same for all these, why are these all optional?

Reply:
I have a note in the PR description 😄 ```Each metric type is configured to be enabled/disabled by a field in the yaml file, denoted by collect_<prefix>_stats (any custom defined queries will have their prefix auto enabled to allow collection), this is to be able to quickly switch on/off useful metrics as well as be able to reduce performance overhead from recursively going through metric types that may have many sub stat nodes.```